### PR TITLE
refactor `EthVideo.tsx` component with chakra UI

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**
## Description
Migrate `EthVideo.tsx` component to Chakra.

Includes refactor of `div` component.

## Related Issue
#6374

### CODE REVIEW

**Review by pettinarip — CHANGES_REQUESTED** [review]
@MahendraBishnoi29 thanks for the PR.

For some reason, I don't see the video. You can check that in the preview deploy generated here https://ethereumorgwebsitedev01-mahendrabishnoi29ethereum00379.gtsb.io/en/eth

image

You can compare it with the production version. https://ethereum.org/en/eth/

I think it is because of how the parent container is positioned. I would recommend avoiding using AspectRatio in this case since I don't see an easy solution. One thing we could do here is to wrap the video tag with a Box and use the same styles we had.

```tsx
<Box as="video" ....>
```

File: src/components/EthVideo.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { useTheme } from "@emotion/react"
 
 import darkVideo from "../assets/ethereum-hero-dark.mp4"
 import lightVideo from "../assets/ethereum-hero-light.mp4"
+import { AspectRatio } from "@chakra-ui/react"
```
Comment (pettinarip) [inline src/components/EthVideo.tsx:6]: import order
```suggestion
import { AspectRatio } from "@chakra-ui/react"

import darkVideo from "../assets/ethereum-hero-dark.mp4"
import lightVideo from "../assets/ethereum-hero-light.mp4"

```

**Comment by MahendraBishnoi29** [comment]
> @MahendraBishnoi29 thanks for the PR.
> 
> For some reason, I don't see the video. You can check that in the preview deploy generated here https://ethereumorgwebsitedev01-mahendrabishnoi29ethereum00379.gtsb.io/en/eth
> 
> image
> 
> You can compare it with the production version. https://ethereum.org/en/eth/
> 
> I think it is because of how the parent container is positioned. I would recommend avoiding using AspectRatio in this case since I don't see an easy solution. One thing we could do here is to wrap the video tag with a Box and use the same styles we had.
> 
> ```tsx
> <Box as="video" ....>
> ```

alright lemme fix it.

**Comment by MahendraBishnoi29** [comment]
@pettinarip I did your suggested changes have a look. 🛠

**Review by pettinarip — APPROVED** [review]
🚀 thanks @MahendraBishnoi29 

@MahendraBishnoi29 be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by pettinarip** [comment]
@all-contributors please add @MahendraBishnoi29 for code

**Pull request merged**